src/app-bar.jsx

```diff
@@ -260,7 +268,7 @@ const AppBar = React.createClass({
         }
 
         menuElementLeft = (
-          <div style={prepareStyles(Object.assign({}, styles.iconButtonStyle))}>
+          <div style={prepareStyles(iconLeftStyle)}>
```

Comment:
`iconRightStyle` will change the style of `menuElementRight`.
`iconLeftStyle` should also be certain so.